content/docs/start/experiments.md

```diff
@@ -21,39 +21,34 @@ the [`example-dvc-experiments`][ede] project.
 
 <details>
 
-### ⚙️ Installing the example project
+### ⚙️ Initializing a project into DVC experiments
 
-These commands are run in the [`example-dvc-experiments`][ede] project. You can
-run the commands in this document after cloning the repository, installing the
-requirements, and pulling the data.
+If you already have a DVC project, that's great. You can start to use `dvc exp`
+commands right away to run experiments in your project. (See the [user's guide]
+for detailed information.) Here, we briefly discuss how to structure an ML
+project into a DVC experiments project with `dvc exp init`.
```

Comment:
same here ... `into` reads strange to me to be honest

Reply:
Updated in ea7c4537